src/io/inputs/model-converter/include/antares/io/inputs/model-converter/convertorVisitor.h

```diff
@@ -62,6 +65,47 @@ class NoConstraintWithThisName final: public std::runtime_error
     }
 };
 
+class AntlrParsingError final: public std::invalid_argument
+{
+public:
+    explicit AntlrParsingError(const std::string& msg):
+        std::invalid_argument(msg)
+    {
+    }
+};
+
+class ErrorListener: public antlr4::BaseErrorListener
+{
```

Comment:
To make it clear that this our error listener, not ANTLR's
```suggestion
class AntaresErrorListener: public antlr4::BaseErrorListener
{
```